index.js

```diff
@@ -25,9 +26,13 @@ const serve = ({endPoint, port, useSSL, accessKey, secretKey, bucketName}) => {
 
     return async function serve(ctx, next) {
         if (ctx.request.method === 'GET') {
-            const fileInBucket = ctx.request.url.split('?')[0];
-            let dataStream;
+            const splittedUrl = ctx.request.url.split('?');
+            const fileInBucket = splittedUrl[0];
+            const queryParams = splittedUrl.length > 1 ? parse(splittedUrl[1]) : {};
+
+            ctx.set('Content-Disposition', `${queryParams.contentDisposition ||  'attachment' }; filename="${queryParams.filename || fileInBucket}"`);
```

Comment:
@stephane-klein If this is set to `attachment` by default files will be downloaded by default

Reply:
@Its-Alex yes I know, but I need to define a default value, `inline` or `attachment`.

Do you prefer `inline` by default?

Reply:
@stephane-klein I think that by default chrome/firefox try to open a file, so I would put `inline` here to keep default behavior, what do you think?

Reply:
@stephane-klein For now we will let it like that